templates/taxi/driver_detail.html

```diff
@@ -23,4 +23,11 @@ <h4>Cars</h4>
       <p>No cars!</p>
     {% endfor %}
   </div>
+  <a style="float: left" href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
```

Comment:
Code Style: It is suggested to add a 'Cancel' button apart from 'Delete' one. The 'Cancel' button will lead to the previous page the user was on.